INSTALL.rst.txt

```diff
@@ -135,50 +109,39 @@ Use the command::
 
   git clone https://github.com/scipy/scipy.git
 
-Before building and installing from git, remove the old installation
-(e.g. in /usr/lib/python3.4/site-packages/scipy or
+If you have an older version of Scipy installed and didn't install that with
+pip, you need to manually uninstall it first (remove from e.g. in
+/usr/lib/python3.4/site-packages/scipy or
 $HOME/lib/python3.4/site-packages/scipy).  Then type::
 
   cd scipy
   git clean -xdf
-  python setup.py install
+  pip install . --user
```

Comment:
It does. I would not recommend to use ``pip`` for development. But this doc is about installing, not developing. So it's for most people a one-time operation (assuming things go well) - and then the benefit of reliable uninstall is more important imho.